module_utils/helpers.py

```diff
@@ -61,12 +61,12 @@ def __init__(self):
     'feedback_enabled',
     'feedback_host',
     'feedback_interval',
-])
+}
 
-memory_size_box_cfg_params = set([
+memory_size_box_cfg_params = {
     'memtx_memory',
     'vinyl_memory',
-])
+}
```

Comment:
Please, explain this change.

Reply:
`set(['a', 'b', 'c'])` first creates a list, and then converts it to a set. `{'a', 'b', 'c'}` immediately creates a set. The IDE also generates a warning when you use the `set(['a', 'b', 'c'])` syntax.

But if you think that the previous version was more readable, then I will return it.

Reply:
I'm not sure that this change should be performed in this patch. You can perform it in separated commit ant merge this PR without squash.

Reply:
Okey, I made two commits